src/design-guidelines/color-system/color-system.md

```diff
@@ -65,30 +105,30 @@ Colors should be used to communicate meaning, and attract attention to things th
 Use only one accent color (product color) and use it for all Lime Elements components in your product. Define this accent color by setting the CSS variable `--lime-primary-color`. We don't recommend using the Lime brand color CSS variables directly in this case, but instead suggest using their equivalent swatch from the color palette. This will ensure that your UI will automatically benefit from dark mode adjustments, while users do not notice the difference.
 
 Things that can be affected by the accent color:
-* In buttons: background color of primary buttons
-* In check boxes: background color of “checked” boxed
-* In radio buttons: background color of “checked” buttons
-* In inputs: the underline visual effect
-* In linear progress: color of the progressed section and what’s left (its background)
-* In many components: the visual styles that indicate the `focused` state (using keyboard)
-* In many components: the visual styles that indicate the `active` or `selected` state
-* etc…
 
-What colors you can optionally change to communicate meaning:
-* In buttons: text color, and also background color of buttons with certain functionalities (delete, restore, etc)
-* In badges: color of the page and its text color
-* In linear progress: color of the progressed section and what’s left (its background)
-* etc…
+-   In buttons: background color of primary buttons
+-   In check boxes: background color of “checked” boxed
+-   In radio buttons: background color of “checked” buttons
+-   In inputs: the underline visual effect
+-   In linear progress: color of the progressed section and what’s left (its background)
+-   In many components: the visual styles that indicate the `focused` state (using keyboard)
+-   In many components: the visual styles that indicate the `active` or `selected` state
+-   etc…
 
-<limel-example-colors-in-components></limel-example-colors-in-components>
+What colors you can optionally change to communicate meaning:
 
+-   In buttons: text color, and also background color of buttons with certain functionalities (delete, restore, etc)
+-   In badges: color of the page and its text color
+-   In linear progress: color of the progressed section and what’s left (its background)
+-   etc…
 
+<limel-example-colors-in-components></limel-example-colors-in-components>
```

Comment:
The colours for the buttons have broken here.

<img width="1173" alt="image" src="https://user-images.githubusercontent.com/154725/211839680-5e67171d-78e9-4fd5-92a0-1b777363ce06.png">

Reply:
⚡